modules/installation-aws-hcp-sd-ami-stream-metadata.adoc

```diff
@@ -0,0 +1,42 @@
+//TODO: Add the module include to the following assemblies
+//TODO: Create related modules for OpenStack (QCOW2) and Bare Metal (ISO)
+
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-aws-hcp-sd-ami-stream-metadata_{context}"]
+= Accessing ROSA AMIs with stream metadata
+
+In {product-title}, _stream metadata_ provides standardized metadata about RHCOS in the JSON format and injects the metadata into the cluster. Stream metadata is a stable format that supports multiple architectures and is intended to be self-documenting for maintaining automation.
+
+You can use the `coreos print-stream-json` sub-command of `openshift-install` to access information about the boot images in the stream metadata format. This command provides a method for printing stream metadata in a scriptable, machine-readable format.
+
+For user-provisioned installations, the `openshift-install` binary contains references to the version of RHCOS boot images that are tested for use with {product-title}, such as the AWS AMI.
+
+.Procedure
+
+To parse the stream metadata, use one of the following methods:
+
+* From a Go program, use the official `stream-metadata-go` library at https://github.com/coreos/stream-metadata-go. You can also view example code in the library.
+
+* From another programming language, such as Python or Ruby, use the JSON library of your preferred programming language.
+
+* From a command-line utility that handles JSON data, such as `jq`:
+
+** Print the current `x86_64` AMI for an AWS region, such as `us-west-1`:
++
+.For x86_64
```

Comment:
I'm not sure what you are trying to say here. Are you trying to say: "To obtain/print the `x86_64` of a specific region, run the following command:"